mod/settings.php

```diff
@@ -504,14 +516,15 @@ function settings_post(App $a) {
 		}
 	}
 
-	if(strlen($err)) {
+	if (strlen($err)) {
 		notice($err . EOL);
 		return;
 	}
 
-	if($timezone != $a->user['timezone']) {
-		if(strlen($timezone))
+	if ($timezone != $a->user['timezone']) {
+		if (strlen($timezone)) {
```

Comment:
This can be condensed in a single condition.

Reply:
Done.